src/components/ImportAccountsDrawer/ImportAccountsDrawer.tsx

```diff
@@ -0,0 +1,249 @@
+import {
+  Button,
+  Drawer,
+  DrawerBody,
+  DrawerCloseButton,
+  DrawerContent,
+  DrawerFooter,
+  DrawerHeader,
+  DrawerOverlay,
+  forwardRef,
+  Skeleton,
+  Switch,
+  Table,
+  TableContainer,
+  Tbody,
+  Td,
+  Tooltip,
+  Tr,
+} from '@chakra-ui/react'
+import type { ChainId } from '@shapeshiftoss/caip'
+import { type AccountId, fromAccountId } from '@shapeshiftoss/caip'
+import type { Asset } from '@shapeshiftoss/types'
+import { useIsFetching, useQueryClient } from '@tanstack/react-query'
+import { useCallback, useEffect, useMemo, useState } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { reactQueries } from 'react-queries'
+import { Amount } from 'components/Amount/Amount'
+import { MiddleEllipsis } from 'components/MiddleEllipsis/MiddleEllipsis'
+import { RawText, Text } from 'components/Text'
+import { useWallet } from 'hooks/useWallet/useWallet'
+import { accountIdToLabel } from 'state/slices/portfolioSlice/utils'
+import {
+  selectFeeAssetByChainId,
+  selectPortfolioCryptoPrecisionBalanceByFilter,
+} from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+export type ImportAccountsDrawerProps = {
+  chainId: ChainId
+  isOpen: boolean
+  onClose: () => void
+}
+
+type TableRowProps = {
+  accountId: AccountId
+  accountNumber: number
+  feeAsset: Asset
```

Comment:
Since we're talking native assets at all times here, I think we could simplify this and rename `asset` to avoid being overly declarative and potentially confusing as a result

Reply:
no probs, will get this done.